contrib/scalar/scalar.c

```diff
@@ -68,6 +76,24 @@ static void setup_enlistment_directory(int argc, const char **argv,
 	}
 
 	setup_git_directory();
+
+	/* Scalar requires the use of a credential helper */
+	setenv("GIT_TERMINAL_PROMPT", "false", 0);
```

Comment:
This seems wrong.. we're never going to allow terminal-based prompting (including in GCM).
GCM checks `GIT_TERMINAL_PROMPT` too!